### PR TITLE
Release 2.0.0: Task 6 - fix a flaky test

### PR DESCRIPTION
**Pull request by CCheSumo**
Looks like we set the timeout on some tests that causing flakiness. In this PR I remove the timeout in these tests.

### CODE REVIEW

**Review by cddude229 — APPROVED** [review]

**Comment by davidcarltonsumo** [comment]
I'm a little nervous about removing it entirely - can we set it to a minute or even 10 minutes instead?  I don't want jobs to just get stuck...

**Comment by CCheSumo** [comment]
Time out set to 5 seconds. That is more than enough to get the test stable. Previous, it was 10ms :(.

File: elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala [hunk 1]
```diff
@@ -111,7 +111,7 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
     "Be able to create an index, index a document, and search it" in {
       indexDocs(Seq(Document("doc1", Map("text" -> "here"))))
       val resFut = restClient.query(index, tpe, new QueryRoot(TermQuery("text", "here"),
-        sortOpt = Some(Seq(SimpleSort("_timestamp", DescSortOrder))), timeoutOpt = None))
+        sortOpt = Some(Seq(SimpleSort("_timestamp", DescSortOrder))), timeoutOpt = Some(50000)))
```
Comment (davidcarltonsumo) [inline elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala]: this one is 50 seconds, did you mean 5 seconds here too?

**Comment by davidcarltonsumo** [comment]
happy to go with 5 seconds and see how it works. (Though our Sumo Jenkins is not the only place where I've needed to use a timeout of a minute, so I won't be shocked if we need to set it to longer than 5 seconds.)

**Review by davidcarltonsumo — APPROVED** [review]
lgtm except for the presumably unintended 50 seconds on one

**Comment by davidcarltonsumo** [comment]
(not that I have anything against 50 seconds as a value!)

**Comment by CCheSumo** [comment]
Meant to be 5 seconds. Updating

**Pull request merged**